packages/keystone/lib/Keystone/index.js

```diff
@@ -155,8 +164,7 @@ module.exports = class Keystone {
     // Deduping here avoids that problem.
     return [
       ...unique(flatten(this.listsArray.map(list => list.getGqlTypes({ skipAccessControl })))),
-      ...unique(flatten(this.listsArray.map(list => list.getGqlQueryTypes()))),
-      ...unique(flatten(this.listsArray.map(list => list.getGqlMutationTypes()))),
+      ...this._extendedTypes,
```

Comment:
We should `unique` these still.
```suggestion
      ...unique(this._extendedTypes),
```

Reply:
Good point 👍